src/mots/cli.py

```diff
@@ -253,6 +257,29 @@ def search(args: argparse.Namespace):
         print(out)
 
 
+def check_for_updates(args: argparse.Namespace) -> None:
+    """
+    Show a message if there is a newer version available.
+
+    This method checks the RSS feed on PyPI.
+    """
+    URL = "https://pypi.org/rss/project/mots/releases.xml"
+    response = requests.get(URL)
+    result = ElementTree.fromstring(response.content)
+    items = result[0].findall("item")
+    versions = [Version(item[0].text) for item in items]
```

Comment:
You might want to add checks for things like `if result`, `if result[0]`, `if items` etc.